scripts/index.js

```diff
@@ -1,25 +1,21 @@
 import "babel/polyfill";
-import btoa from "btoa";
+import { EventEmitter } from "events";
 import React from "react";
+
 import App from "./components/App";
-import { Store } from "./store";
-import Kinto from "kinto";
+import Auth from "./auth";
+import Flux from "./flux";
+import Store from "./store";
 
-// Take username from location hash (With default value):
-const user = window.location.hash = (window.location.hash.slice(1) || "public-demo");
-const userpass64 = btoa(user + ":s3cr3t");
 
-// Use Mozilla demo server with Basic authentication:
-const server = "https://kinto.dev.mozaws.net/v1";
-const kinto = new Kinto({
-  remote: server,
-  // Make sure local data depend on current user.
-  dbPrefix: user,
-  // Provide authentication header.
-  headers: {Authorization: "Basic " + userpass64}
-});
+const DEFAULT_SERVER = "https://kinto.dev.mozaws.net/v1";
 
-const store = new Store(kinto, "items");
+const events = new EventEmitter();
+const auth = new Auth(events);
+const store = new Store("items", events);
+const flux = new Flux({auth, store}, events);
 
+React.render(<App flux={flux}/>, document.getElementById("app"));
 
-React.render(<App store={store}/>, document.getElementById("app"));
+// Start with default server.
+events.emit('action:configure', {server: DEFAULT_SERVER});
```

Comment:
Should we replace this by a 'action:start' from main `App::componentDidMount()` ?